src/lib/components/evm-to-cell/evm-to-cell-create.tsx

```
@@ -21,17 +22,35 @@ interface EvmToCellCreateProps {
-      <CustomIcon boxSize={3} color="primary.main" name="contract-address" />
+const EvmToCellCreateContracts = ({
+  contracts,
+}: {
+  contracts: HexAddr20[];
+}) => {
+  const { data: evmVerifyInfos, isLoading: isEvmVerifyInfosLoading } =
+    useEvmVerifyInfos(contracts);
+
+  return contracts.map((contract) =>
+    isEvmVerifyInfosLoading ? (
+      <Spinner boxSize={4} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)